packages/sdk/src/contracts/prebuilt-implementations/marketplace.ts

```diff
@@ -438,6 +438,26 @@ export class Marketplace implements UpdateableNetwork {
     await this.contractWrapper.multiCall(encoded);
   }
 
+  /**
+   * returns the minimum bid a user can place to outbid the previous highest bid
+   * @param listingId - the listing id of the auction
+   */
+  public async getMinimumNextBid(
+    listingId: BigNumberish,
+  ): Promise<BigNumberish> {
+    const currentAuction = this.auction;
+    const currentBidBuffer = await this.getBidBufferBps();
+    const minimumBid: BigNumberish = await this.getMinimumNextBid(listingId);
```

Comment:
this seems like it's calling itself recursively? I assume you meant to call a function to determine the *absolute* minimum bid for the listing instead?

Reply:
Hi @jnsdls, thanks for the feedback. I completely missed this. will push another commit to handle this

Reply:
Hi @jnsdls , I have made a new commit. I honestly not 100% with the minimum value but I assumed that if there's no current auction yet and no winning bid, means nothing has made a bid yet, so basically, I have to get the reservePrice and add bid buffer to it.